app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -77,6 +77,9 @@ span.input-group-btn .hidden-xs {display:inline !important;}
 span.btn a {
   text-decoration: none;
   color: #333333;
+  // make sure the link inside span covers the whole area of the button
+  padding: 8px;
+  margin: -8px
 }
 
```

Comment:
I always solve this by giving the `<a/>` element `display: inline-block;` styling. The enclosing element should also have `block` or `inline-block display`

Generally, use `padding` instead of `margin` between an element and its enclosing element